### DISCUSSION TITLE
Unable to send the traces to gateway url using auto instrumentation - The request could not be executed. Full error message: timeout when i use HTTP/Protobuf

### DISCUSSION THREAD

**Discussion by ramanjaneyagupta** [open-telemetry/opentelemetry-java-instrumentation#8182] [Q&A]
Hi When i am trying to send the traces using java auto instrumentation jar to the gateway collector(a set of collectors running behind the VIP).

and it is giving following error : 
ERROR io.opentelemetry.exporter.internal.okhttp.OkHttpExporter-Failed to export spans.
29
The request could not be executed. Full error message: timeout

Configuration: 
java -javaagent:/home/rpasumar/agent/opentelemetry-javaagent_1.24.jar\
-Dotel. traces. exporter=otlp\
-Dotel.exporter.otlp.traces.endpoint=<gateway-url>:4318/v1/traces
-Dotel.exporter. otlp. traces.protocol=http/protobuf
-Dotel.service.name=petclinic 2
-Dserver.port=8081
-jar target/spring-petclinic-2.6.0-SNAPSHOT.jar

NOTE: if i have the collector(agent mode) installed on same vm and send the traces to that local collector, from local collector to gateway i didnt see any issue.but directly sending it to gateway it is giving above issue.

**Comment by trask** [comment]
hi @ramanjaneyagupta, is this a duplicate of #8181?

**Reply by ramanjaneyagupta** [reply]
Similar but this is for http protobuf and other is for grpc ..
